client/components/SongBarEntry.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SongBarEntry = ({ height, bar }) => (<span
+  className="player-songbar-bar"
```

Comment:
I would consider bringing the span tag on a new row (or was this an airbnb suggestion?)

Reply:
Will address this in revisions!